platform/jewel/samples/ide-plugin/src/main/kotlin/org/jetbrains/jewel/samples/ideplugin/ComponentShowcaseTab.kt

```diff
@@ -271,12 +271,12 @@ private fun RowScope.ColumnOne() {
 
                 2 -> {
                     WarningDefaultBanner("This is a warning banner in Compose")
-                    WarningInlineBanner("This is a warning banner in Compose")
+                    WarningInlineBanner("This is a warning banner in Compose", actions = null)
                 }
 
                 else -> {
                     InformationDefaultBanner("This is an information banner in Compose")
-                    InformationInlineBanner("This is an information banner in Compose")
+                    InformationInlineBanner("This is an information banner in Compose", actions = null)
```

Comment:
@rock3r I had to add this to make the code compile again. Any suggestions on how to work around this to prevent breaking on the client?